routes/lambdaRouter.js

```diff
@@ -0,0 +1,97 @@
+const router = require('express').Router()
+const knex = require('knex')
+
+const knexConfig = require('../knexfile')
+
+const db = knex(knexConfig.development)
+
+router.get('/', async (req, res) => {
+    try {
+        const cohorts = await db('cohorts')
+        res.status(200).json(cohorts)
+    } catch(error) {
+        res.status(500).json(error)
+    }
+})
+
+router.get('/:id', async (req, res) => {
+    const id = req.params.id
+    try {
+        const cohort = await db('cohorts')
+        .where({ id })
+        if (cohort.length > 0) {
+            return res.status(200).json(cohort)
+        } else {
+            return res.status(404).json({ message: "The provided ID does not correspond with an existing cohort" })
+        }
+    } catch(error) {
+        console.log(error)
+        res.status(500).json(error)
+    }
+})
+
+router.get('/:id/students', async (req, res) => {
+    const id = req.params.id
+    try {
+        const cohort = await db('cohorts')
```

Comment:
You don't necessarily need this query. Finding the students in the students table by using the 'cohort_id' column is sufficient.

Reply:
However the check to see if the cohort exists has some utility to it.